modules/manual-configuration-image-registry-ibm.adoc

```diff
@@ -12,9 +12,10 @@ You can integrate {product-title} with IBM Cloud Container Registry.
 
 .Procedure
 . On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
-. Under the *Images* section, select *Google Container Registry*.
+. Under the *Image Integrations* section, select *Google Container Registry*.
```

Comment:
Should this step refer to IBM Cloud Container Registry instead of Google Container Registry?